CONTRIBUTING.md

```diff
@@ -1,126 +1,177 @@
 
-* There is possibility to execute terratest suite over the real clusters.
-For this you need to override the set of test settings as in example below.
-```sh
-PRIMARY_GEO_TAG=af-south-1 \
-SECONDARY_GEO_TAG=eu-west-1 \
-DNS_SERVER1=a377095726f1845fb85b95c2afef8ac0-9a1a10f24e634e28.elb.af-south-1.amazonaws.com \
-DNS_SERVER1_PORT=53 \
-DNS_SERVER2=a873f5c83be624a0a84c05a743d598a8-443f7e0285e4a28f.elb.eu-west-1.amazonaws.com \
-DNS_SERVER2_PORT=53 \
-GSLB_DOMAIN=test.k8gb.io \
-K8GB_CLUSTER1=arn:aws:eks:af-south-1:<aws-account-id>:cluster/k8gb-cluster-af-south-1 \
-K8GB_CLUSTER2=arn:aws:eks:eu-west-1:<aws-account-id>:cluster/k8gb-cluster-eu-west-1 \
-make terratest
-```
+- Acceptance terratest suite is located [here](https://github.com/AbsaOSS/k8gb/tree/master/terratest) and executable by `make terratest` target.
+  These tests are updated only if the change is substantial enough to affect the main end-to-end flow.
 
-- [Debugging](#debugging)
+- There is a possibility to execute the terratest suite over the real clusters.
+For this, you need to override the set of test settings as in the example below.
+  ```sh
+  PRIMARY_GEO_TAG=af-south-1 \
+  SECONDARY_GEO_TAG=eu-west-1 \
+  DNS_SERVER1=a377095726f1845fb85b95c2afef8ac0-9a1a10f24e634e28.elb.af-south-1.amazonaws.com \
+  DNS_SERVER1_PORT=53 \
+  DNS_SERVER2=a873f5c83be624a0a84c05a743d598a8-443f7e0285e4a28f.elb.eu-west-1.amazonaws.com \
+  DNS_SERVER2_PORT=53 \
+  GSLB_DOMAIN=test.k8gb.io \
+  K8GB_CLUSTER1=arn:aws:eks:af-south-1:<aws-account-id>:cluster/k8gb-cluster-af-south-1 \
+  K8GB_CLUSTER2=arn:aws:eks:eu-west-1:<aws-account-id>:cluster/k8gb-cluster-eu-west-1 \
+  make terratest
+  ```
 
-### Debugging
+## Debugging
 
-Delve debugger needs to be installed first. Follow the [installation instructions](https://github.com/go-delve/delve/tree/master/Documentation/installation) for specific platforms from Delve's website.
+1. Install Delve debugger first. Follow the [installation instructions](https://github.com/go-delve/delve/tree/master/Documentation/installation) for specific platforms from Delve's website.
 
-1. Run the following script
+2. Run the following script
 
-```shell script
-> make debug-local
-```
+    ```shell script
+    > make debug-local
+    ```
 
-2. Attach debugger of your favourite IDE to port `2345`.
+3. Attach debugger of your favorite IDE to port `2345`.
 
-### Code style
+## Code style
 
-The coding style suggested by the Golang community is used in k8gb project. See the [style doc][golang-style-doc] for details.
+k8gb project is using the coding style suggested by the Golang community. See the [golang-style-doc](https://github.com/golang/go/wiki/CodeReviewComments) for details.
 
 Please follow this style to make k8gb easy to review, maintain and develop.
-
 Run `make lint` to automatically check if your code is compliant.
 
-### Format of the commit message
+## Pull Request message
```

Comment:
I think Pull Request description and commit message are not mutually exclusive.
```suggestion
## Pull Request description and commit message
```

Reply:
@ytsarev Then we should probably separate them, because current template is not something we follow for commit messages, only for PR messages. Commit messages are usually following this template:
```
<Desctribe what and why has changed>
```
Do we want commit messages to strictly follow PR message template with additional message body and footer, wdyt?

Reply:
commit messages can be lightweight.

Quite a frequent scenario when a single commit message is equal PR description.

Another scenario is when several minimalistic commit messages are going to PR description as bullet points of what was done.